src/display/svg.js

```diff
@@ -1107,11 +1107,10 @@ SVGGraphics = (function SVGGraphicsClosure() {
 
     fill: function SVGGraphics_fill() {
       var current = this.current;
-      if (current.element) {
```

Comment:
This `if` statement should not be removed; if was added in https://github.com/mozilla/pdf.js/commit/4e8f7070da236c6a3a8572ff141d6a4ce231bb94 to fix a bug.